python3/unittest/test_usb_scan.py

```diff
@@ -107,65 +104,70 @@ def test_log(m):
 
 
 class TestUsbScan(unittest.TestCase):
-
     def setUp(self):
-        try:
-            self.work_dir = tempfile.mkdtemp(prefix="test_usb_scan")
-        except:
-            raise
+        self.work_dir = tempfile.mkdtemp(prefix="test_usb_scan")
 
     def tearDown(self):
         shutil.rmtree(self.work_dir, ignore_errors=True)
 
     @nottest
-    def test_usb_common(self, moc_devices, moc_interfaces, moc_results,
-                        path="./scripts/usb-policy.conf"):
-        import usb_scan
+    def test_usb_common(
+        self, moc_devices, moc_interfaces, moc_results, path="./scripts/usb-policy.conf"
+    ):
+        usb_scan = get_module("usb_scan", "../libexec/usb_scan.py")
+
         mock_setup(usb_scan, moc_devices, moc_interfaces, path)
 
         devices, interfaces = usb_scan.get_usb_info()
 
+        usb_scan.log_list(devices)
+        usb_scan.log_list(interfaces)
+
         pusbs = usb_scan.make_pusbs_list(devices, interfaces)
 
         # pass pusbs in json to XAPI
         self.assertEqual(sorted(pusbs), sorted(moc_results))
 
     @nottest
```

Comment:
```suggestion
```
I think when renaming the prefix of the affected functions not be `test_` but `verify_`, the `@nottest` decorator can be fully removed from all utility functions in this test, pylint disable is then not needed and, the decorator itself can be removed as well.